src/Controller/CommandLineController.ts

```diff
@@ -0,0 +1,47 @@
+import yargs from 'yargs'
+
+export interface JunknetArguments {
+	makefile: string
+	docker_image: string
+	target: string | null
+}
+
+/**
+ * Interpret a list of arguments, returning the relevant ones in the f
+ *
+ * @param argv - the invoking process's command-line arguments.
+ * @returns JunknetInterface object containing option values.
```

Comment:
Don't put the return type name in the docstring since it's redundant (also, in this case not consistent).